pkgs/development/compilers/bluespec/default.nix

```diff
@@ -18,20 +18,26 @@
 , gmp-static
 , verilog
 , asciidoctor
-, tex }:
+, tex
+, which }:
 
 let
   ghcWithPackages = ghc.withPackages (g: (with g; [ old-time regex-compat syb split ]));
 
 in stdenv.mkDerivation rec {
   pname = "bluespec";
-  version = "2021.07";
+  version = "2022.01";
 
   src = fetchFromGitHub {
     owner = "B-Lang-org";
     repo = "bsc";
     rev = version;
-    sha256 = "0gw8wyp65lpkyfhv3laazz9qypdl8qkp1j7cqp0gv11592a9p5qw";
+    sha256 = "sha256-ivTua3MLa8akma3MGkhsqwSdwswYX916kywKdlj7TqY=";
+  };
+
+  yices-src = builtins.fetchurl {
+    url = "https://github.com/B-Lang-org/bsc/releases/download/${version}/yices-src-for-bsc-${version}.tar.gz";
+    sha256 = "sha256:0w8r509dmzlws265bl3vnvb4lv5a16raj1kn2aw22rs5hlh74bkv";
```

Comment:
```suggestion
    sha256 = "0w8r509dmzlws265bl3vnvb4lv5a16raj1kn2aw22rs5hlh74bkv";
```

Reply:
Switched to pkgs.fetchurl, which also switched the hash to the normal `sha256-` form.